### PR TITLE
Changed local.txt to grab edx-i18n-tools from PyPI rather than Github

### PR DESCRIPTION
**Pull request by Dillon-Dumesnil**
@bderusha @clintonb Please review

### CODE REVIEW

**Comment by clintonb** [comment]
👍

File: {{cookiecutter.repo_name}}/requirements/local.txt [hunk 1]
```diff
@@ -4,5 +4,5 @@
 
 django-debug-toolbar
 docker-compose >= 1.7.1
-git+https://github.com/edx/i18n-tools.git@v0.3.1#egg=i18n_tools==0.3.1
+edx-i18n-tools >= 0.3.3
```
Comment (nedbat) [inline {{cookiecutter.repo_name}}/requirements/local.txt]: What's the recommendation about version numbers?  Usually IDA's like to pin exact versions.

Comment (Dillon-Dumesnil) [reply {{cookiecutter.repo_name}}/requirements/local.txt]: @nedbat I was basing the style here on the way docker-compose was being used. Perhaps Clinton can comment on it more when he is back online

Comment (clintonb) [reply {{cookiecutter.repo_name}}/requirements/local.txt]: The version number can be removed in both cases.

**Comment by iivic** [comment]
@clintonb @mjfrey Confirm merge.

**Comment by clintonb** [comment]
Update .gitignore and include the English .po files.

**Comment by iivic** [comment]
Review once more.

**Comment by clintonb** [comment]
👍

**Comment by iivic** [comment]
@clintonb I can't generate translations for this repo because it has no repo name. The translations committed are from a new project created following the readme instructions. Am I missing something?

**Comment by clintonb** [comment]
@iivic just remove them from this repo. The person who uses this template will generate them.

**Comment by iivic** [comment]
Removed .po file. Review once more before merge.

**Comment by clintonb** [comment]
👍

**Pull request merged**